### ISSUE TITLE
Problem with CheckHelpCommand

### ISSUE THREAD

**Issue by gertjaap** [mit-dci/lit#231]
In lit-af, we added a `CheckHelpCommand` utility function to check the required number of parameters and print out the command format/description when you specify `-h`. However, in this new setup the control flow of the main function is not ended when someone enters `-h` because it will only break on an error. Consequence is that command that only require 1 parameter will continue to execute trying to parse `-h` as the first parameter.

Most obvious example is when you execute `close -h`:

```
lit-af# close -h
entered command: close -h
close <channel idx>
Cooperatively close the channel with the given index by asking
the other party to finalize the channel pay-out.
See also: break
close error: strconv.Atoi: parsing "-h": invalid syntax
```

**Comment by Varunram** [comment]
Nice catch, I'll take this one up.

**Comment by Varunram** [comment]
Fixed in #250
